packages/server/dataloader/hydrateRedisDoc.ts

```diff
@@ -8,9 +8,6 @@ const hydrators = {
     return publicTemplates
   },
   User: (user: IUser) => {
```

Comment:
-1 I think the function is wrongly typed and we still want to ensure the fields are converted to `Date`

Reply:
Actually I don't quite understand this function at all, can you elaborate a bit more? 🙏

Reply:
I don't know the exact context the function is used, only what's in this file. In line 19 we can see it's used to convert parsed JSON object into the proper type. Since JSON does not have a date type, that's why the conversion was here.
The argument type of this User conversion function is incorrect since the input does not have the correct type for the date fields yet.

Reply:
Ohhh I see... it's the type of argument `IUser` gets me confused. Thanks!